README.md

```diff
@@ -235,6 +235,30 @@ rainbow table attacks. There are multiple ways to do so, listed by order of prec
 
 The salt should be of the same encoding as the associated data to hash.
 
+### Sanitize hash
+
+> _Support: introduced in version 1.6.0_
+
+You can sanitize a hash before creation and querying. This might be useful in case you would like to find a User with the name of `François ` with a query input of `francois`.
+
+There are several sanitize options:
+
+```
+/// @encryption:hash(email)?sanitize=lowercase  <- lowercase hash
+/// @encryption:hash(email)?sanitize=uppercase  <- uppercase hash
+/// @encryption:hash(email)?sanitize=trim       <- trim start and end of hash
+/// @encryption:hash(email)?sanitize=spaces     <- remove spaces in hash
+/// @encryption:hash(email)?sanitize=diacritics <- remove diacritics like ç or é in hash
+```
+
+You can also combine the sanitize options:
+
+```
+/// @encryption:hash(email)?sanitize=lowercase&sanitize=trim&sanitize=trim&sanitize=diacritics
```

Comment:
**suggestion**: This is a bit verbose to read, what do you think about grouping sanitizers under a single key, with comma-separated values ?

Eg:
```prisma
/// @encryption:hash(email)?sanitize=lowercase,trim,diacritics
```

This way, I can see more clearly that there isn't a duplication of `trim`, unlike the original query string.

Also, adding a note saying that the order is irrelevant could be useful (I only realized that the sanitizers are commutative after reading the code where they are applied).

Reply:
That was my first thought as well, but under the hood it is using `URLSearchParams`, and the documentation of it told me you cannot do comma seperated inputs ([https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#duplicate_search_parameters](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#duplicate_search_parameters)). I found some hacky ways online, but my preference was to stay close to the spec.

Reply:
Yes this is more spec-compliant. In [`nuqs`](https://nuqs.47ng.com/docs/parsers#arrays), I've opted to go for comma (or any user-provided character) separated values for arrays. It helps with readability, but requires a `.split(',')` when reading the value.